build/dependencies.yaml

```diff
@@ -27,10 +27,8 @@ dependencies:
   - name: "docker"
     version: 19.03
     refPaths:
-    - path: cmd/kubeadm/app/util/system/docker_validator.go
+    - path: vendor/k8s.io/system-validators/validators/docker_validator.go
       match: latestValidatedDockerVersion
-    - path: cmd/kubeadm/app/util/system/docker_validator_test.go
-      match: ServerVersion
```

Comment:
@yastij did you mention that we should no longer verify the unit test too?

this is fine in general, as the system-validators repo now has a PR blocking job that runs the unit tests.

Reply:
Yes, this should be fine